tests/test_hrrr_preprocessor.py

```diff
@@ -1,86 +1,64 @@
-import os
-import shutil
-import unittest
-
 import pandas as pd
 
+from tests.RME_test_case import RMETestCase
 from weather_forecast_retrieval.hrrr import HRRR
 from weather_forecast_retrieval.hrrr_preprocessor import HRRRPreprocessor
 
 
-class TestHRRRPreprocessor(unittest.TestCase):
-    """Test cropping HRRR files"""
-
-    def setUp(self):
-        """
-        Test the retrieval of existing data that will be passed to programs
-        like SMRF
-        """
-
-        self.bbox_crop = [-116.9, 42.9, -116.5, 43.2]
```

Comment:
Didn't realize these were in almost all the tests! Good call on making a base class for these.